src/Library/Lod/LodReader.h

```diff
@@ -30,11 +27,31 @@ class LodReader final {
     bool exists(const std::string &filename) const;
     Blob read(const std::string &filename);
 
+    void cd(const std::string &filename);
```

Comment:
Please revisit the discussions we had about the `LodReader` interface.

Adding `cd` to `LodReader` is effectively the same as adding `iterator` as a field in `vector`. We should avoid this as it almost always leads to problems down the line.